birdhouse/optional-components/weaver/conf.extra-service.d/weaver.conf.template

```diff
@@ -0,0 +1,22 @@
+
+    # NOTE:
+    #   Redirect to internal network of twitcher with Weaver root endpoint and alias allows to set
+    #   the same 'magpie' permissions on the 'weaver' service defined by "WEAVER_MANAGER_NAME".
+    #   This allows verification of the same service user/group permissions references regardless
+    #   whether the *shortcut* Weaver endpoint, the alias or the explicit 'twitcher' proxy route is used.
+    location /${WEAVER_CONFIG} {
+        # redirect to secured EMS/ADES
+        return 302 https://${PAVICS_FQDN}${TWITCHER_PROTECTED_PATH}/${WEAVER_MANAGER_NAME};
```

Comment:
We do not need to specify port here and assume that it is on :80?

Reply:
It is https route, do default should be 443